### DISCUSSION TITLE
Sandboxed opens node shells on Windows

### DISCUSSION THREAD

**Discussion by notflip** [taskforcesh/bullmq#2008] [Q&A]
When I use the sandboxed processors, when a job is spawned, an empty terminal-ish window opens with Node as the title, Is there a way to have these child_process windows hidden?

**Comment by manast** [comment]
No window should be opened, in which OS is this happening?

**Reply by notflip** [reply]
Microsoft Server 2019
![Screenshot 2023-06-23 at 10 25 58](https://github.com/taskforcesh/bullmq/assets/4146384/a330a26b-1ca0-4db2-b18b-fa71c3a3c896)

**Reply by manast** [reply]
It must be some Node issue on that particular platform, I have never seen something like that. You could try using worker threads instead: https://api.docs.bullmq.io/interfaces/WorkerOptions.html#useWorkerThreads
